pkg/monitor/dynkeepalived.go

```diff
@@ -430,7 +430,7 @@ func KeepalivedWatch(kubeconfigPath, clusterConfigPath, templatePath, cfgPath st
 				// if the path doesn't exist then RemoveAll returns nil
 				log.WithFields(logrus.Fields{"path": iptablesFilePath}).WithError(err).Error("Failed to remove file")
 			}
-			newConfig, err := config.GetConfig(kubeconfigPath, clusterConfigPath, "/etc/resolv.conf", apiVips, ingressVips, 0, 0, 0, config.ClusterLBConfig{})
+			newConfig, err := config.GetConfig(kubeconfigPath, clusterConfigPath, "/etc/resolv.conf", apiVips, ingressVips, 0, 0, 0, config.ClusterLBConfig{}, "")
```

Comment:
Unless there is a reason not to, we should keep passing the `platform` parameter all around instead of placeholder with empty value

Reply:
Same case as https://github.com/openshift/baremetal-runtimecfg/pull/345/files#r2035844045. I am trying not to introduce too many changes within code that is run only on on-prem environments.

Reply:
Done.